docs/browser/events.md

```diff
@@ -0,0 +1,50 @@
+<!--- Hugo front matter used to generate the website version of this page:
+linkTitle: Events
+--->
+
+# Semantic conventions for browser events
+
+**Status**: [Development][DocumentStatus]
+
+This document defines semantic conventions for browser (web) instrumentations
+that emit events.
+
+## WebVital Event
+
+<!-- semconv event.browser.web_vital -->
+<!-- NOTE: THIS TEXT IS AUTOGENERATED. DO NOT EDIT BY HAND. -->
+<!-- see templates/registry/markdown/snippet.md.j2 -->
+<!-- prettier-ignore-start -->
+<!-- markdownlint-capture -->
+<!-- markdownlint-disable -->
+
+**Status:** ![Development](https://img.shields.io/badge/-development-blue)
+
+The event name MUST be `browser.web_vital`.
+
+This event describes the website performance metrics introduced by Google, See [web vitals](https://web.dev/vitals).
+
+**Body fields:**
+
+| Body Field  | Type | Description  | Examples  | [Requirement Level](https://opentelemetry.io/docs/specs/semconv/general/attribute-requirement-level/) | Stability |
+|---|---|---|---|---|---|
+| `delta` | double | The delta between the current value and the last-reported value. | `0.2` | `Required` | ![Development](https://img.shields.io/badge/-development-blue) |
```

Comment:
I briefly looked over web vitals docs, but I didn't see this delta concept, can you explain a bit why it's needed / how someone would interpret a high or low delta value? thanks!

Reply:
The Delta concept is introduced here: https://github.com/GoogleChrome/web-vitals?tab=readme-ov-file#report-only-the-delta-of-changes. I will add a link in the brief/notes